Explicitly note that snapshots are excluded from full refresh (#9831)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/full_refresh.md

| If set to `false` | The resource _never_ performs a full refresh, regardless of whether you pass the `--full-refresh` flag in the dbt command. |
| If set to `none` or omitted | The resource follows the behavior of the `--full-refresh` flag. If the flag is used, the resource will perform a full refresh; otherwise, it will not. |

> :::info Snapshots are excluded from full refresh

Reviewer: ```suggestion
:::info Snapshots ignore full refresh
```

> The `full_refresh` config and the `--full-refresh` flag only apply to incremental models and seeds. [Snapshots](/docs/build/snapshots) still run as normal, but they ignore `--full-refresh`: running a command such as `dbt build --full-refresh` that includes a snapshot node will not drop or recreate the snapshot table, so existing snapshot history is preserved.

Reviewer: ```suggestion
Snapshots ignore both the `full_refresh` config and the `--full-refresh` flag. A command such as `dbt build --full-refresh` or `dbt snapshot --full-refresh` that includes a snapshot node runs the snapshot as normal — it won't drop or recreate the snapshot table, so existing snapshot history is preserved.
```
